src/foam/box/BoxRegistry.js

```diff
@@ -70,6 +70,13 @@ foam.CLASS({
       ]
     },
     {
+      name: 'getRegisteredNames',
+      returns: 'Array',
+      code: function() {
+        return Object.keys(this.registry);
+      }
+    },
+    {
```

Comment:
Having `DedicatedWorkerRegistry` keep track of registrations is not sufficient. This is because we pass `localRegistry` to functions, which make registrations that do not go through `DedicatedWorkerRegistry` (but technically belonging to `DedicatedWorkerRegistry`).

Thus, name conflicts would still be possible if we only relied on a map property that is updated upon register and unregister.